docs/sources/flow/getting-started/opentelemetry-to-lgtm-stack.md

```diff
@@ -0,0 +1,308 @@
+---
+title: OpenTelemetry to LGTM stack
+weight: 350
+---
+
+# OpenTelemetry to LGTM stack
+
+In the [previous guide][], we learnt how to ingest [OpenTelemetry][] data, and export it to a different destination. In this one, we will continue from there but send it to a Loki, Mimir and Tempo stack.
```

Comment:
nit: Do we want to say Mimir here or Prometheus? The guide below talks a lot about Prometheus instead and in grafana cloud it is labeled as prometheus for me. Alternatively, we may want to more explicitly talk about Mimir in the Prometheus section for newcomers if we do want to keep it this way up here.

Reply:
Why not Prometheus? :)

Reply:
Added Mimir